src/AdminAcceptanceTests.Actions/Pages/UserAccountsDashboard.cs

```diff
@@ -93,6 +163,67 @@ public bool ExpectedUserHasDisabledFlag(string username)
             return row.FindElements(Objects.Pages.UserAccountsDashboard.DisabledAccountFlag).Count == 1;
         }
 
+        public bool ErrorSummaryDisplayed()
+        {
+            return Driver.FindElements(Objects.Pages.CreateBuyerUser.ErrorSummary).Count > 0;
+        }
+
+        public bool ErrorMessagesDisplayed()
+        {
+            return Driver.FindElements(Objects.Pages.UserAccountsDashboard.ErrorMessages).Count > 0;
+        }
+
+        public bool RemoveLinkDisplayed()
+        {
+            return Driver.FindElements(Objects.Pages.UserAccountsDashboard.RemoveLink).Count > 0;
+        }
+
+        public bool RemoveButtonDisplayed()
+        {
+            return Driver.FindElements(Objects.Pages.UserAccountsDashboard.RemoveButton).Count > 0;
+        }
+
+        public void ClickRemoveButton()
+        {
+            Driver.FindElement(Objects.Pages.UserAccountsDashboard.RemoveButton).Click();
+        }
+
+        public string ClickRelatedRemoveButton(int index = 0)
+        {
+            Wait.Until(d => NumberOfRadioButtonsDisplayed() > index);
+            var element = Driver.FindElements(Objects.Pages.UserAccountsDashboard.RadioButton)[index];
+            element.Click();
+            Wait.Until(d => bool.Parse(d.FindElements(Objects.Pages.UserAccountsDashboard.RadioButton)[index].GetProperty("checked")));
+            return element.GetAttribute("name");
```

Comment:
This method looks like it's selecting a radio button rather than clicking the remove button for whatever it requires. Is this correct or should it be doing something different?

Reply:
will sort now

Reply:
removed